dockers/docker-config-engine-buster/Dockerfile.j2

```diff
@@ -12,6 +12,10 @@ RUN apt-get update         && \
         python-dev            \
         python3-dev           \
         apt-utils             \
+{%- if CONFIGURED_ARCH == "armhf" or CONFIGURED_ARCH == "arm64" %}
+        libxslt-dev           \
+        libz-dev              \
```

Comment:
wonder why we do not need to specify these for amd64?

Reply:
lxml-4.6.2-cp37-cp37m-manylinux1_x86_64.whl is directly downloaded in amd64 whereas in arm this is built from lxml-4.6.2.tar.gz